src/xdsl/dialects/llvm.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+from dataclasses import dataclass
+
+from xdsl.irdl import *
```

Comment:
Could you import manually the list of things you want? We are currently removing the `from _ import *` from xdsl.